app/experimenter/jetstream/client.py

```diff
@@ -261,7 +262,19 @@ def get_experiment_data(experiment):
                     errors_by_metric[metric_slug] = []
                 errors_by_metric[metric_slug].append(err)
             else:
-                errors_experiment_overall.append(err)
+                try:
+                    analysis_start_time = datetime.fromisoformat(
+                        experiment_metadata.get("analysis_start_time", "")
+                        if experiment_metadata is not None
+                        else ""
+                    )
+                    timestamp = datetime.fromisoformat(err.get("timestamp", ""))
```

Comment:
So in both of these cases we pass an empty string to `fromisoformat`, but why not flip it the other way around and say `analysis_start_time` and `timestamp` are `None` unless each are an actual iso datetime in which case parse and compare?  I see this locally:

```
>>> datetime.datetime.fromisoformat("")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: Invalid isoformat string: ''
```

Reply:
The problem is determining in advance (and with certainty) whether the timestamps are going to be valid without passing them to `fromisoformat`, which means we're risking an exception (`ValueError` if string is invalid, `TypeError` if not a string) either way. This seemed like the cleanest way to handle that. The default `""` in the `get()` calls are probably superfluous though.